src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -555,12 +566,31 @@ bool MatchesClientId(Task<IIdentity> identity)
             }
             catch (Exception exc)
             {
+                if (exc is KubernetesException)
+                {
+                    var kExc = (KubernetesException)exc;
+                    ConsoleEx.WriteLine($"Kubenetes Status: {kExc.Status}");
+                }
+
+                if (exc is WebSocketException)
+                {
+                    var wExc = (WebSocketException)exc;
+                    ConsoleEx.WriteLine($"WebSocket ErrorCode: {wExc.WebSocketErrorCode}"); 
+                }
+
+                if (exc is HttpOperationException)
+                {
+                    var hExc = (HttpOperationException)exc;
+                    ConsoleEx.WriteLine($"HTTP Response: {hExc.Response.Content}");
+                }
+
                 if (!(exc is OperationCanceledException && cts.Token.IsCancellationRequested))
                 {
                     ConsoleEx.WriteLine();
                     ConsoleEx.WriteLine($"{exc.GetType().Name}: {exc.Message}", ConsoleColor.Red);
+                    ConsoleEx.WriteLine(exc.StackTrace, ConsoleColor.Red);
                 }
```

Comment:
Do we always want to print a stack trace? I know that can sometimes be a security issue.